_includes/live-demos/premiumskinsandicons-fabric/index.js

```diff
@@ -2,8 +2,7 @@ tinymce.init({
   selector: 'textarea#premiumskinsandicons-fabric',
   skin: 'fabric',
   content_css: [
```

Comment:
This could be simplified to just `content_css: 'fabric'` now if you wanted.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/8bc36cb9e57a4c890e01fa0a70ebc3484a22a7bb